### PR TITLE
[14.0][IMP]sale_commission_geo_assign: add required group to docs

### PR DESCRIPTION
**Pull request by PicchiSeba**

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
LGTM

**Comment by francesco-ooops** [comment]
@pedrobaeza @eLBati doc improvement, can it be merged?

File: sale_commission_geo_assign/README.rst [hunk 1]
```diff
@@ -55,6 +55,8 @@ Go to Sales > Agents > open a record: in tab "Agent information", set geographic
 
 When using server action, this agent will be assigned to customers whose address match the set criteria.
 
+A user must be in the 'Contact Creation' group in order to access this action
```
Comment (pedrobaeza) [inline sale_commission_geo_assign/README.rst]: This should go to CONFIGURE.rst instead.

**Comment by PicchiSeba** [comment]
@pedrobaeza Moved to CONFIGURE.rst

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**